pkg/kv/kvserver/replica_application_cmd.go

```diff
@@ -100,6 +100,7 @@ func (c *replicatedCmd) getStoreWriteByteSizes() (writeBytes int64, ingestedByte
 	if c.Cmd.WriteBatch != nil {
 		writeBytes = int64(len(c.Cmd.WriteBatch.Data))
 	}
+	// TODO(msbutler): do we need extra logic for external ssts?
 	if c.Cmd.ReplicatedEvalResult.AddSSTable != nil {
```

Comment:
I believe the answer here is no: `writeBytes` and `ingestedBytes` should be 0 for an external file.

Reply:
Going with 0, to keep the accounting consistent with AddSStable: we only increment for data, not metadata.